### PR TITLE
Fix Bug 1399277, post launch content updates for dev firstrun

### PR DESCRIPTION
**Pull request by schalkneethling**
## Description
Post launch copy updates for developer firstrun page

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1399277

## Testing
Ensure no regressions
Ensure all new strings are correctly wrapped in conditionals and for l10n

### CODE REVIEW

**Comment by peiying2** [comment]
Could you put the test URL in here? Also content wise, this is identical with PR5203 from l10n perspective.

**Comment by schalkneethling** [comment]
> Could you put the test URL in here? Also content wise, this is identical with PR5203 from l10n perspective.

@peiying2 Yes, they both referenced this document:
https://docs.google.com/document/d/1OboHPRVEKOgbyEbEzWWg5zMBSCBg8QJIPPTI_Nsmfo8/edit

**Comment by schalkneethling** [comment]
@peiying2 I am pushing it to the following server:
https://bedrock-demo-schalk.us-west.moz.works/

This will have both whatsnew and firstrun updates.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 1]
```diff
@@ -68,13 +68,22 @@ <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
         <div class="section-body">
           <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
 
-          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" alt="">
+          <a href="http://firefox-dev.tools/" rel="external"><img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" {% if l10n_has_tag('quantum-firstrun-whatsnew') %}alt="{{ _('Learn more about Firefox developer tools') }}"{% endif %}></a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: I think this needs an `else` to fall back to an empty alt, or just put the whole thing inside the attribute like `alt="{% if %}...{% endif %}"` so it falls back to empty.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 2]
```diff
@@ -84,7 +93,7 @@ <h2 class="section-title"><span>{{ _('Innovative Features') }}</span></h2>
         <div class="section-body">
           <h3 id="cssgrid" class="body-title">{{ _('Master CSS Grid') }}</h3>
 
-          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-cssgrid-ani.gif') }}" alt="">
+          <a href="https://mozilladevelopers.github.io/playground/" rel="external"><img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-cssgrid-ani.gif') }}" {% if l10n_has_tag('quantum-firstrun-whatsnew') %}alt="{{ _('Learn more about CSS Grid') }}"{% endif %}></a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Same here, empty alt as the fallback.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 3]
```diff
@@ -68,13 +68,22 @@ <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
         <div class="section-body">
           <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Should we link these titles as well? That might need some CSS tweaks.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 4]
```diff
@@ -126,73 +135,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-inspector.svg') }}" alt="">
             <h3 class="name">{{ _('Inspector') }}</h3>
-            <p class="desc">{{ _('Inspect and refine code to build pixel-perfect layouts.') }}</p>
+            <p class="desc">
+              {{ _('Inspect and refine code to build pixel-perfect layouts.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/docs/Tools/Page_Inspector" rel="external">{{ _('Learn more about Page Inspector') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: This nests a `p` inside a `p`. We could style the links with `display:block` themselves without the extra paragraph (same for all of them).

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 5]
```diff
@@ -126,73 +135,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
...
           </li>
 
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-memory.svg') }}" alt="">
             <h3 class="name">{{ _('Memory') }}</h3>
-            <p class="desc">{{ _('Find memory leaks and make your application zippy.') }}</p>
+            <p class="desc">
+              {{ _('Find memory leaks and make your application zippy.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/en-US/docs/Tools/Memory" rel="external">{{ _('Learn more about Memory Tools') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Remove `en-US` from the URL so MDN's locale detection can take over.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html [hunk 6]
```diff
@@ -126,73 +135,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
...
           </li>
 
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-scratchpad.svg') }}" alt="">
             <h3 class="name">{{ _('Scratchpad') }}</h3>
-            <p class="desc">{{ _('Edit, write, run and execute JavaScript in real time.') }}</p>
+            <p class="desc">
+              {{ _('Edit, write, run and execute JavaScript in real time.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/en-US/docs/Tools/Scratchpad" rel="external">{{ _('Learn more about Scratchpad') }}</a></p>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Should remove `en-US` here too.

File: bedrock/firefox/templates/firefox/developer-quantum-firstrun.html (hunk 4 above)
Comment (schalkneethling) [reply bedrock/firefox/templates/firefox/developer-quantum-firstrun.html]: Doh! Completely missed that.

**Comment by schalkneethling** [comment]
Thanks for the review @craigcook ~ updated

**Pull request merged**